projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -223,7 +228,10 @@ export class IgxTimePickerComponent implements
     * An accessor that returns the label of cancel button.
     */
     get cancelButtonLabel(): string {
-        return this._cancelButtonLabel || this.resourceStrings.igx_time_picker_cancel;
+        if (this._cancelButtonLabel === null) {
```

Comment:
Do you actually need to change this code?

Reply:
Same as above.